cmd/kubeturbo/app/kubeturbo_builder.go

```diff
@@ -33,9 +36,10 @@ const (
 )
 
 // VMTServer has all the context and params needed to run a Scheduler
+// TODO: leaderElection is disabled now because of dependency problems.
 type VMTServer struct {
 	Port            int
-	Address         net.IP
+	Address         string
```

Comment:
I think we still need to make it as `net.IP` and use [`net.ParseIP`](https://golang.org/pkg/net/#ParseIP) to enforce the correct format to prevent any other random string.